install.sh

```diff
@@ -76,6 +77,69 @@ function backup_append() {
 	echo -e "\033[0;32mBash-it template has been added to your $CONFIG_FILE\033[0m"
 }
 
+function check_for_backup() {
```

Comment:
I'm not going to road block since this in the install script, but it'd be nice to start making all functions be prefixed with `_bash-it` or something consistent and namespaced.

Reply:
added this prefix to all functions in `install.sh`